packages/terra-tabs/src/Tabs.jsx

```diff
@@ -169,35 +178,44 @@ class Tabs extends React.Component {
       React.cloneElement(contentItem, { isLabelHidden: isIconOnly || this.state.isLabelTruncated })
     ));
 
-    const collapsibleTabs = (
-      <CollapsibleTabs
-        activeKey={activeKey || this.state.activeKey}
-        activeIndex={this.getActiveTabIndex()}
-        onChange={this.handleOnChange}
-        onTruncationChange={this.handleTruncationChange}
-        variant={variant}
-      >
-        {clonedPanes}
-      </CollapsibleTabs>
-    );
+    const header = (() => {
```

Comment:
Can we rename this func to something like getTabs() to be representative of what it is returning?

Reply:
or could we convert this to be something like:
```js
...
let tabContent = collapsibleTabs;

 if ([DependentViewport.PARENT, DependentViewport.WINDOW].includes(responsiveTo)) {
    ...
     tabContent (
          <ResponsiveElement
            tiny={collapsedTabs}
            small={collapsibleTabs}
            responsiveTo={responsiveTo}
      />
    )
 }

....

return (
      <ContentContainer
        {...customProps}
        className={tabsClassNames}
        fill={fill}
        header={tabContent}
      >
...
```

Reply:
I have renamed `header` ➡️ `tabContent` - https://github.com/cerner/terra-framework/pull/968/commits/e293226fe48790135e5279fb37bfa4a6a0f4fb35.